tests/data/channel/normalize.py

```diff
@@ -0,0 +1,37 @@
+"""Test channel normalize."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+import WrightTools as wt
+from WrightTools import datasets
+
+
+# --- test ----------------------------------------------------------------------------------------
+
+
+def test_LDS821():
+    p = datasets.BrunoldrRaman.LDS821_514nm_80mW
+    data = wt.data.from_BrunoldrRaman(p)
+    data.signal.normalize()
+    assert data.signal.null == 0.
+    assert data.signal.max() == 1.
+    data.close()
+
+
+def test_wigner():
+    p = datasets.COLORS.v2p2_WL_wigner
+    data = wt.data.from_COLORS(p)
+    data.ai0.normalize()
+    assert data.ai0.null == 0.
+    assert data.ai0.max() == 1.
+    data.close()
+
+
```

Comment:
Test for signed data normalization?

Reply:
yeah OK

believe it or not, I don't think I have any signed datasets in WrightTools

Reply:
actually, I'm not sure what behavior I want for signed datasets